recipe/meta.yaml

```diff
@@ -28,17 +28,19 @@ requirements:
   run:
     - anytree >=2.8.0
     - autograd >=1.6.2
```

Comment:
```suggestion
```

We dropped `autograd` from our dependencies: https://github.com/pybamm-team/PyBaMM/blob/v24.5/pyproject.toml